README.md

```diff
@@ -50,9 +50,9 @@ allprojects {
 }
 ```
 
-# Dependency breakdown starting in Arrow 0.9.0
+# Dependency breakdown starting in Arrow 0.9.1
```

Comment:
This was correct as it was since 0.9.0 already had the changes mentioned.

Reply:
We're missing the single package version tho